### PR TITLE
ORDERS-6362: [add] is_tax_inclusive_pricing field in orders response

### PR DESCRIPTION
**Pull request by jpajar**
https://bigcommercecloud.atlassian.net/browse/ORDERS-6362

## What changed?

* Added  is_tax_inclusive_pricing field in orders response

Swagger Proof:

https://github.com/user-attachments/assets/86560ebf-3fc7-482d-93b3-4190699b8c96

### CODE REVIEW

File: reference/orders.v2.oas2.yml
Comment (donald-nguyen-bc) [inline reference/orders.v2.oas2.yml:1]: please don't commit the formating changes

btw pls include testing proof with the change displayed correctly in StopLight Studio, and in online swagger editor https://editor.swagger.io/

Comment (bc-traciporter) [reply reference/orders.v2.oas2.yml:1]: HI @donald-nguyen-bc 
Can I put the addition of the new field in a new PR? I don't know how to merge this PR without merging all the formatting changes. 

I attached a screenshot of the new field shown in the swagger editor. We are no longer using Stoplight. Everything is in GitHub.
Screenshot 2024-09-16 at 4 30 03 PM

Comment (WillemHoman) [reply reference/orders.v2.oas2.yml:1]: If dev-docs team want to change the formatting then we will take their lead I think right @donald-nguyen-bc.

@jpajar  It is good practice to have formatting changes be in a separate commit or PR to the actual change which needs review to make it easier for the reviewers.

Comment (jpajar) [reply reference/orders.v2.oas2.yml:1]: @WillemHoman yep apologies, I had format on save on but I removed it since. 

This PR no longer has the formatting changes.

@donald-nguyen-bc I included swagger proof in the PR description

**Comment by WillemHoman** [comment]
Also need to add it to the `order_Resp` @jpajar

**Comment by jpajar** [comment]
@WillemHoman it's added iin this line https://github.com/bigcommerce/docs/pull/525/files#:~:text=%275.3100%27-,is_tax_inclusive_pricing%3A%20false,-items_total%3A%204

**Comment by WillemHoman** [comment]
> @WillemHoman it's added iin this line https://github.com/bigcommerce/docs/pull/525/files#:~:text=%275.3100%27-,is_tax_inclusive_pricing%3A%20false,-items_total%3A%204

There needs to be a text description added to [components.schemas.order_RespOnly.properties](https://github.com/bigcommerce/docs/blob/2edcbea4dee9ce2a37ee6055fa36bf3700158329/reference/orders.v2.oas2.yml#L4018) that gets rendered in this section
Screenshot 2024-09-17 at 2 10 31 PM

File: reference/orders.v2.oas2.yml
Comment (donald-nguyen-bc) [reply reference/orders.v2.oas2.yml]: @jpajar pls show proof for the `schema` instead of `example`.

image

Btw, would be nice to check response of the `POST /v2/orders` as well, it should be updated, but just want to make sure.

Comment (jpajar) [reply reference/orders.v2.oas2.yml]: Schema
image

v2 API:
image

Let me know if any other screenshots are needed! @WillemHoman  @donald-nguyen-bc

**Review by WillemHoman — APPROVED** [review]
Thanks @jpajar

File: reference/orders.v2.oas2.yml
Comment (donald-nguyen-bc) [inline reference/orders.v2.oas2.yml:3921]: seems like we don't need it here, otherwise it will appear in the `POST /v2/orders` body:

post_body

**Comment by donald-nguyen-bc** [comment]
btw, pls wait for devdoc team approval before merging

File: reference/orders.v2.oas2.yml
Comment (donald-nguyen-bc) [inline reference/orders.v2.oas2.yml:4140]: Probably this description would be more meaningful: 

```
Indicate whether the order's base prices are inclusive of tax. 

If true, the base prices are inclusive of tax, and the values of `subtotal_inc_tax`, `shipping_cost_inc_tax`, `handling_cost_inc_tax`, `wrapping_cost_inc_tax` and `total_inc_tax` are not estimated but actual values and can be reliable for accounting purposes.

If false, the base prices are exclusive of tax, and the values of `subtotal_ex_tax`, `shipping_cost_ex_tax`, `handling_cost_ex_tax`, `wrapping_cost_ex_tax` and `total_ex_tax` are not estimated but actual values and can be reliable for accounting purposes.
```

@WillemHoman @bc-vincent-zhao @BC-SEven WDYT about this description, any more info we can add ?

Comment (BC-SEven) [reply reference/orders.v2.oas2.yml:4140]: We could also update the descriptions of those other properties directly to reference back to this one

**Pull request merged**
